.mypy.ini

```diff
@@ -27,3 +27,9 @@ ignore_missing_imports = True
 
 [mypy-pytest]
 ignore_missing_imports = True
+
+[mypy-PIL.*]
+ignore_missing_imports = True
```

Comment:
Turns out there is an effective requirement on Pillow as a function in the iqr-search app space uses it to open image files (`smqtk_iqr/web/search_app/modules/iqr/iqr_search.py:427`). Please add that back in as a primary dependency and update the poetry lock file.